.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  env: {
+    node: true,
+    es6: true,
+    es2020: true,
+    jest: true,
+  },
+  globals: {
+    setupDefaultMocks: "readonly",
+  },
+  root: true,
+  extends: ["eslint:recommended", "prettier"],
```

Comment:
Do we need to add prettier as a plugin?

Reply:
No, adding it as a plugin would allow us to run prettier is a linter rule but I added a new format script to run prettier followed by eslint - this has the benefit of fixing formatting errors automatically instead of throwing eslint warnings for formatting errors.